Adds note about dev mode toggle (#1560)
odoo/documentation: content/applications/general/developer_mode.rst

Go to :menuselection:`Settings --> Activate the developer mode`.

.. note::

>    You need to have at least one app installed for this menu option to appear.

Reviewer: Hi! Can we change this line to: 

`Note: At least one application needs to be installed for the developer menu option to appear.`

Let's remove the familiar second person language and make it less personal.

Reviewer: @javl Are you okay with the above changes? If so, can you update the RST and we will get the doc updated :) Thanks, Tim

>    You need to have at least one app installed for this menu option to appear.

Reviewer: ```suggestion
   At least one application needs to be installed for the developer menu option to appear.
```
